nextflow.config

```diff
@@ -71,7 +71,9 @@ params {
     arg_deeparg_numalignmentsperentry   = 1000
 
     arg_skip_abricate                   = false
-    arg_abricate_db                     = 'card'
+    arg_abricate_db                     = 'ncbi'
```

Comment:
Just for my understanding: ABRicate downloads several databases, but uses only one? Why did we put 'card' and now switch to 'ncbi' as default? Because it is the tools' default?

Reply:
Thats what i understood too. JAsmin used 'card' for her analyses, but i checked and NCBI has more ARG hits and its is the default. I asked @jasmezz and we agreed we keep 'ncbi' as the default for now.